Utils/alerts.py

```diff
@@ -52,7 +61,87 @@ def convert_dataframe_to_bytes(dataframe):
         RawMessage={"Data": message.as_string()},
     )
 
-    print(response)
-
+    logger.info(response)
+
+
+class SlackMessaging:
+    SLACK_TOKEN = os.getenv('SLACK_APP_TOKEN')
+    SLACK_CHANNEL = os.getenv('SLACK_CHANNEL')
```

Comment:
I think this probably should just be a parameter instead. No need for a env var unless we can only post one channel which I'm hoping is not the case.